sig-docs/annual-report-2021.md

```diff
@@ -0,0 +1,125 @@
+    
+    Quite small (usually 5-10 people) and generally we try to make people feel comfortable to give their opinion.
+    Meeting notes are up to date. We have recording of the meetings uploaded.
+    
+    We have a new sub-project, it's localization sub-project, where we conduct monthly meeting 
+    attended by all people from the localizatiin team. This sub-porjcet focusing on sharing any concerns,
+    tips, or toolings that may be can be reused by other team as well.
+
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired?
+Are OWNERS files up to date in these areas?**
+
+Yes it is.
+
+**How does the group get updates, reports, or feedback from Working Groups? Are there any springing up or being retired? 
+Are OWNERS files up to date in these areas?**
+
+Yes it is.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+<!-- TODO -->
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+Yes, they are.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We don't have an official way of measuring membership.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+PR are generally open for anyone to review. We have [PR wrangler rotation](https://github.com/kubernetes/website/wiki/PR-Wranglers).
+In general each of the approver will taking the rotation. Each approver participates in the PR wrangler rotation. Approvers are supported during their rotation by other reviewers and approvers.
+
+For technical reviews, other than SIG Docs Tech Leads, we also notify respective SIGs with necessary knowledge of the technical aspect of the content. SIG Docs receives contributions from community members representing different SIGs. During the PR review process, SIG Docs reaches out to other SIGs when a pull request requires technical review.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? 
+What are some activities that the group does to encourage this? 
+What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+There is no official onboarding process. However, the SIG is supportive of new contributors,
+willing to walk through the contribution process and eager to see contributions completed.
+Issues are created regularly and triaged. Issues are also created specifically for new 
+contributors. There are issues labelled for the different contribution areas (web development, 
+blog, first time contributors, task, concept, API, etc) which helps new contributors 
+identify how to get started.
+
+From time to time we use the weekly meeting Zoom call to help new contributors get onboard.
+
+**What programs do you participate in for new contributors?**
+
+We have an in person session during the KubeCon.
```

Comment:
We used to host doc sprints, as things get "back to normal" it would be good to call out as an activity. Not sure if it needs to land in this report or not... The doc sprints acted as a way to onboard new contributors as well as tackle a lot of open issues.